lib/big_query/resource.ex

```diff
@@ -50,6 +50,23 @@ defmodule BigQuery.Resource do
     end
   end
 
+
+    @spec delete(String.t, any, [{String.t, String.t}], [timeout: integer]) :: {:ok, response} | {:error, String.t}
+    def delete(url, body \\ nil, headers \\ [], opts \\ [timeout: 120_000]) do
```

Comment:
The the delete function doesn't use a body parameter at all, so the right thing to do here would be just remove it, as opposed to defaulting to `nil`. Remember to update the spec, too.